xamarin/Xamarin.Forms
# [iOS] Fix infinite scroll issue in CollectionView using groups

### Description of Change ###

Fix infinite scroll issue in CollectionView using groups on iOS.

![fix8383](https://user-images.githubusercontent.com/6755973/135871388-4c1c525e-b980-4fab-88a6-0be06a2c685d.gif)

### Issues Resolved ### 

- fixes #8383 

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 8383. When scrolling down, new items should automatically load when reaching the end of the list, like on Android.

### PR Checklist ###
